test/Formik.test.tsx

```diff
@@ -542,6 +542,17 @@ describe('<Formik>', () => {
         expect(validate).not.toHaveBeenCalled();
       });
 
+      it('setValues takes a function which can patch values', () => {
+        const { getProps } = renderFormik();
+
+        getProps().setValues((values: Values) => ({
```

Comment:
is the Type of `values` inferrable here? if so, avoid manually typing it

Reply:
Looks like it is: I just pushed a fix.